pkg/util/iptables/save_restore.go

```diff
@@ -52,7 +52,7 @@ func GetChainLines(table Table, save []byte) map[Chain]string {
 		} else if strings.HasPrefix(line, "#") {
 			continue
 		} else if strings.HasPrefix(line, ":") && len(line) > 1 {
-			chain := Chain(strings.SplitN(line[1:], " ", 2)[0])
+			chain := Chain(line[1:strings.Index(line, " ")])
```

Comment:
That's actually a good point.
@thockin - this is parsing itables, so in theory we have no guarantee over the contents right?
If we can't assume the lines contains space, this will not work (I mean we need to check against -1 value).